packages/command/src/sfdxFlags.ts

```diff
@@ -35,24 +35,26 @@ function option<T>(
 
 export namespace flags {
   export type Array<T> = Option<T[]> & { delimiter?: string; map?: (val: string) => T };
-  export type Any<T> = Partial<OclifFlags.IFlag<T>> & Describable;
+  export type Any<T> = Partial<OclifFlags.IFlag<T>> & SfdxProperties;
   export type BaseBoolean<T> = Partial<IBooleanFlag<T>>;
-  export type Boolean<T> = BaseBoolean<T> & Describable;
-  export type Builtin = { type: 'builtin'; description?: string; longDescription?: string };
+  export type Boolean<T> = BaseBoolean<T> & SfdxProperties;
+  export type Builtin = { type: 'builtin' } & Partial<Describable>;
   export type DateTime = Option<Date>;
+  export type Deprecatable = { deprecated?: { message: string; version: string } };
```

Comment:
This PR only sets the ability to legitimately attach deprecation info to flag definitions.  It doesn't actually use them, it just prevents us from losing them in the toolbelt migration to oclif.  We'll need some kind of separate story about porting deprecations in the oclif branch.

Reply:
W-5712253